utilities/gitopsctl/cmd/download-job.go

```diff
@@ -0,0 +1,63 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	downloadjob "github.com/redhat-appstudio/managed-gitops/utilities/gitopsctl/implementations/download-job"
+	"github.com/spf13/cobra"
+)
+
+// jobCmd represents the job command
+var jobCmd = &cobra.Command{
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) == 0 {
+			return fmt.Errorf("missing URL argument: use --help flag for details")
+		} else if len(args) != 1 {
+			return fmt.Errorf("too many arguments: use --help flag for details")
+		}
+
+		url := args[0]
+		if !strings.HasPrefix(url, "https://") {
```

Comment:
Question, will this always be the case?  I guess we can change this when the need arises.

Reply:
Yup, it should always be the case.